digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/AssetDataImporterSpec.scala

```diff
@@ -207,6 +233,60 @@ class AssetDataImporterSpec extends FunSuite with Matchers {
     }
   }
 
+  test("if the geometry is a loop then it should be updated") {
```

Comment:
When I run this test it crashes in AssetDataImporter.updateRoadAddressesGeometry on line 296. I guess it is because transaction is already started and it is trying to start another transaction.

Reply:
Now it fails in error:
java.sql.SQLSyntaxErrorException: ORA-01722: invalid number